nss-day-cohort-48/giffygram-altavista-giffygram
# Ken nav

#### Completed Functionality For The NavBar



1. Added file `NavBar.js` to `./src/scripts/nav/NavBar.js` directory.
1. Modified file `NavBar.js` to include `NavBar` and functionality.
2. Modified file `messages.css` to include `.inbox with a top margin` and functionality.
3. Modified file `post.css` to include `.post__feed with a top margin` and functionality.

   ​

#### Steps to Review

1. Checkout this branch locally.
   ```
   git fetch --all
   git checkout ken-nav
   ```
2. Open a new Terminal and open the giffygram repo (make sure it's named 'giffygram')
3. Run `./serve-giffygram` to serve the code.
4. Test app functionality.
   > Make sure the navbar is navigating the page correctly and there are no issues
   > Make sure the css styling fits with the page and the navbar
5. View code file.
   > Confirm file modifications are present as indicated above.
   > Confirm no unused code or extraneous comments exist.
